tardis/__init__.py

```diff
@@ -12,13 +12,20 @@
 # ----------------------------------------------------------------------------
 
 import sys
+import warnings
 
 # ----------------------------------------------------------------------------
 
-from astropy import physical_constants, astronomical_constants
+if ("astropy.units" in sys.modules) or ("astropy.constants" in sys.modules):
+    warnings.warn(
+        "Astropy is already imported externally. Astropy should be imported"
+        " after TARDIS."
```

Comment:
Sounds like a quick fix for me now.

Reply:
I don't think that there is any real 'fix' for this as the Exception comes from Astropy itself. This is just a guard such that TARDIS continues running and doesn't leave the user with a potentially cryptic Exception. At the worst, one uses a different version of the Astropy constants. This will probably mess with e.g. test results, but I think a warning is enough and the code doesn't need to raise an Exception at this point. 

On a side note, ``ruff`` formatting and checking sorts the Astropy import before the TARDIS import, so this is bound to occur at some point.